scripts/release/group_test.py

```diff
@@ -141,5 +132,28 @@ def run_pytest_jobs(pytest_jobs: dict, my_env: dict):
             yaml.safe_dump(cover_yaml, yf)
 
     if failed_jobs:
-        logger.log_warning(f"{len(failed_jobs)} jobs failed. {failed_jobs}.")
+        logger.warning(f"{len(failed_jobs)} jobs failed. {failed_jobs}.")
         sys.exit(1)
+
+
+if __name__ == '__main__':
+    logger.info("Start running group tests")
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input-dir", required=True, type=Path, help="dir path of tests folder")
+    parser.add_argument("-g", "--test-group", required=True, type=str, help="test group name")
+    parser.add_argument("-s", "--subscription", required=True, type=str, help="Subscription ID")
+    parser.add_argument("-r", "--resource-group", required=True, type=str, help="Resource group name")
+    parser.add_argument("-w", "--workspace-name", required=True, type=str, help="Workspace name")
+    parser.add_argument("-c", "--coverage-report", required=False, type=Path, help="Path of coverage report yaml")
+    parser.add_argument("-v", "--version-suffix", required=False, type=str,
+                        help="version suffix which will be used to identify the asset id in tests")
+    args = parser.parse_args()
+    tests_dir = args.input_dir
+    test_group = args.test_group
+    subscription_id = args.subscription
+    resource_group = args.resource_group
+    workspace = args.workspace_name
+    coverage_report = args.coverage_report
+    version_suffix = args.version_suffix
+
+    group_test(tests_dir, test_group, subscription_id, resource_group, workspace, coverage_report, version_suffix)
```

Comment:
Same thing here. Let's make it simpler:
```suggestion
    group_test(args.input_dir, args.test_group, args.subscription, args.resource_group, args.workspace_name,
               args.coverage_report, args.version_suffix)
```